src/fastga/models/aerodynamics/components/digitization/compute_k_prime_single_slotted.py

```diff
@@ -73,27 +73,30 @@ def compute_partials(self, inputs, partials, discrete_inputs=None):
         flap_angle = inputs["flap_angle"]
         chord_ratio = inputs["chord_ratio"]
 
+        flap_angle_clipped = np.clip(flap_angle, -0.08, 79.32)
```

Comment:
If I was petty, I'd point out that ths could probably be changed to
```suggestion
        flap_angle_clipped = np.clip(flap_angle, 0.0, 80.00)
```